.gitignore

```diff
@@ -4,5 +4,4 @@ npm-debug.log
 **/npm-debug.log
 src/client/lib
 src/server/config/keys.js
-src/server/db/index.js
```

Comment:
You have a merge conflict in .gitignore. 
Rebase, fix the merge conflict and push again.